src/mgmt/srv_pool.c

```diff
@@ -439,14 +439,17 @@ ds_mgmt_pool_list_cont(uuid_t uuid, d_rank_list_t *svc_ranks,
  *					Note: ranks may be empty (i.e., *ranks->rl_nr may be 0).
  *					The caller must free the list with d_rank_list_free().
  * \param[in][out]	pool_info	Query results
+ * \param[in][out]	pool_layout_ver	Pool global version
+ * \param[in][out]	upgrade_layout_ver Latest pool global version this pool might be upgraded
```

Comment:
Nit, indentation of the comment should be fixed

Reply:
Thanks for good suggestion, since this is a backport PR, i will push a new pr for master to address your and Kris's good comments.

Thanks!